download_debian_keyring/download_debian_keyring.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+set -eu
+
+export LANG=C
+
+KEYSERVER="keyserver.ubuntu.com "
+
+TEMPDIR=$(mktemp -d)
+SUITE="stable"
+if [ -n $1 ]; then
+  SUITE="$1"
+fi
+
+BASEURL="http://ftp.debian.org/debian"
+if [ -n $2 ]; then
+  BASEURL="$2"
+fi
+
+GPG="gpg --homedir $TEMPDIR --quiet"
```

Comment:
Curious to why we're not using `\gpg` here.

Reply:
I had it in there actually. Due to the `"` I used `\\` but then it did not work for some reason :thinking: